src/Entity.cs

```diff
@@ -88,17 +104,16 @@ private List<Entity> Build(dynamic response)
         private Entity ParseEntity(dynamic response)
         {
             Entity entity = null;
-            if (response["entity"] == "payment")
-            {
-                entity = new Payment();
-            }
-            else if (response["entity"] == "refund")
+            
+            string responseEntity = (string) response["entity"];
+
+            if (Entities.ContainsKey(responseEntity) == true)
             {
-                entity = new Refund();
+                entity = Entities[responseEntity];
             }
-            else if (response["entity"] == "order")
+            else if (response.Property("deleted") != null)

```

Comment:
@sunnyagg Request has to return a list of entities. Therefore, I think it would be better if we can return a deleted token for this api call. Perhaps a good idea would be to manually set entity.Attributes.entity = "token" for this particular case. What do you think?